sanctuary/sanctuary-types.d.ts

```diff
@@ -54,6 +54,7 @@ type NonZeroFiniteNumber = number; // - Infinity, -Infinity, 0, -0, NaN
 type ValidNumber = number; // - NaN
 
 export interface Functor<F> {
+  map<U>(fn: (f: F) => U): Functor<U>;
```

Comment:
Ah, I didn't realise Sanctuary expects Fantasy Land methods to be prefixed with `fantasy-land/`? Is that part of Fantasy Land?